Do not keep updating the skipMessage if the ad is not skippable (#223)
bitmovin/bitmovin-player-ui: CHANGELOG.md

### Added
- `UIContainerConfig.userInteractionEventSource` to allow tracking of user interaction events (which toggle the visibility of certain components like the `ControlBar`) on a custom element

### Changed

> - Hidden `adSkipButton` is not constantly updated anymore if a non skippable ad is playing

Reviewer: `` Avoid unnecessary updating of hidden `AdSkipButton` `` (component names always start with an uppercase letter)

> - Hidden `SkipMessage` is not constantly updated anymore if an ad is playing and is not skippable

Reviewer 2: ```suggestion
- Hidden `adSkipButton` is not constantly updated anymore if a non skippable ad is playing
```
